packages/rancher-vsphere/rancher-vsphere-cpi/generated-changes/patch/templates/daemonset.yaml.patch

```diff
@@ -0,0 +1,20 @@
+--- charts-original/templates/daemonset.yaml
++++ charts/templates/daemonset.yaml
+@@ -23,7 +23,7 @@
+         tier: control-plane
+     spec:
+       {{- if .Values.cloudControllerManager.nodeSelector }}
+-      nodeSelector:
++      nodeSelector: {{ include "linux-node-selector" . | nindent 8 }}
```

Comment:
if nodeSelectors aren't supplied, the affinity rules were fine.

However, if nodeSelectors are supplied, we should still work on a Windows cluster.